R/plotConceptCounts.R

```diff
@@ -0,0 +1,62 @@
+#' Title
+#'
+#' @param result A summarised_result object (output of summariseInObservation).
+#' @param facet Columns to face by. Formula format can be provided. See possible
+#' columns to face by with: `visOmopResults::tidyColumns()`.
+#' @param colour Columns to colour by. See possible columns to colour by with:
+#' `visOmopResults::tidyColumns()`.
+#'
+#' @return A ggplot showing the concept counts
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' library(OmopSketch)
+#' library(dplyr)
+#'
+#' con <- DBI::dbConnect(duckdb::duckdb(),
+#'                       dbdir = CDMConnector::eunomia_dir())
+#' cdm <- CDMConnector::cdm_from_con(con,
+#'                                   cdm_schem = "main",
+#'                                   write_schema = "main")
+#'
+#' results <- summariseConceptCounts(cdm,
+#'          conceptId = list(poliovirus_vaccine = c(40213160)))
+#'
+#' plotConceptCounts(result)
+#'
+#' PatientProfiles::mockDisconnect(cdm)
+#' }
+plotConceptCounts <- function(result,
+                              facet = NULL,
+                              colour = NULL){
+  # initial checks
+  omopgenerics::validateResultArgument(result)
+
+  # subset to results of interest
+  result <- result |>
+    visOmopResults::filterSettings(.data$result_type == "summarise_concept_counts")
+  if (nrow(result) == 0) {
+    cli::cli_abort(c("!" = "No records found with result_type == summarise_in_observation"))
+  }
+
+  # check only one estimate is contained
+  estimate <- unique(result$estimate_name)
```

Comment:
why this check on estimate_name, if you are going to use only "record_count" inn the plot?